Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.hxx

```diff
@@ -581,9 +581,9 @@ namespace AzToolsFramework
 
         enum class InspectorLayout
         {
-            ENTITY = 0,     // All selected entities are regular entities
-            LEVEL,          // The selected entity is the level prefab container entity
-            INVALID         // Other entities are selected alongside the level prefab container entity
+            Entity = 0,                 // All selected entities are regular entities.
+            OpenPrefabContainerEntity,  // The selected entity is the prefab container entity for the focused prefab.
```

Comment:
Since you seem to be using 'openContainerEntity' for overrides in your other PR, should this be called 'FocusedPrefabContainerEntity'? The comment also indicates that its for the focused prefab only

Reply:
Done, I split this in two
```
            Level,                          // The selected entity is the prefab container entity for the level prefab, or the slice level entity.
            ContainerEntityOfFocusedPrefab, // The selected entity is the prefab container entity for the focused prefab.
```